### PR TITLE
Bugfix/add timer to refresh token

### PR DESCRIPTION
**Pull request by RomanKuk**
now token successfully refreshes

### CODE REVIEW

**Comment by RomanKuk** [comment]
The problem of incorrect page still exists((

**Comment by RomanKuk** [comment]
the user will be logged out if something is wrong

File: frontend/src/app/modules/work-space/work-space/work-space.component.ts [hunk 1]
```diff
@@ -44,6 +44,10 @@ export class WorkSpaceComponent extends BaseComponent implements OnInit {
   }
 
   getGroups() {
+    if (this.user === null) {
```
Comment (VoBilyk) [inline frontend/src/app/modules/work-space/work-space/work-space.component.ts:47]: An empty user shouldn't be here

**Review by VoBilyk — CHANGES_REQUESTED** [review]

File: frontend/src/app/modules/work-space/work-space/work-space.component.ts (hunk 1 above)
Comment (RomanKuk) [reply frontend/src/app/modules/work-space/work-space/work-space.component.ts:47]: no, it was, the problem was that user was null

Comment (RomanKuk) [reply frontend/src/app/modules/work-space/work-space/work-space.component.ts:47]: I checked it in this way: register, drop db and refresh page, the user was null as a result, so log out

**Pull request closed without merging**